src/main/frontend/dicts.cljs

```diff
@@ -303,7 +303,8 @@
         :help/shortcut-page-title "Keyboard shortcuts"
 
         :pdf/copy-ref "Copy ref"
-        :pdf/copy-text "Copy text"}
+        :pdf/copy-text "Copy text"
+        :pdf/linked-ref "Goto annotation"}
```

Comment:
Go to?

Reply:
Hah, just align with `Copy` letters size.